src/libs/actions/BankAccounts.js

```diff
@@ -751,7 +753,9 @@ function setupWithdrawalAccount(data) {
                 }
             } else {
                 if (response.jsonCode === 666 || response.jsonCode === 404) {
-                    error = response.message;
+                    // Since these specific responses can have an error message in html format with richer content, give priority to the html error.
+                    error = response.htmlMessage || response.message;
+                    htmlError = Boolean(response.htmlMessage);
```

Comment:
I think you should rename this to something like `isErrorHTML`

Reply:
updated!